### PR TITLE
[test] Delete `@org_tensorflow//tensorflow/core:lib`

### PR DESCRIPTION
**Pull request by ManfeiBai**

### CODE REVIEW

File: third_party/xla_client/BUILD [hunk 1]
```diff
@@ -360,7 +359,6 @@ cc_library(
         "@com_google_absl//absl/types:span",
         "@org_tensorflow//tensorflow/compiler/xla:statusor",
         "@org_tensorflow//tensorflow/compiler/xla:types",
-        "@org_tensorflow//tensorflow/core:lib",
```
Comment (ddunl) [inline third_party/xla_client/BUILD:363]: Change this to @org_tensorflow//tensorflow/tsl/platform:hash

Comment (ManfeiBai) [reply third_party/xla_client/BUILD:363]: tried and passed BUILD, but failed on CPU test

**Comment by JackCaoG** [comment]
Seems like it failed the CPP test build 
```
//third_party/xla_client:async_task_test                        FAILED TO BUILD
```

try to repo locally might be easier.

**Comment by ManfeiBai** [comment]
this PR is old and will continue work in https://github.com/pytorch/xla/pull/5202

**Pull request closed without merging**